DXRMapFixups/DeusEx/Classes/DXRFixupM03.uc

```diff
@@ -353,6 +353,8 @@ function PreFirstEntryMapFixes()
             }
         }
 
+        SetAllLampsState(true, false, true); // alex isn't in his office
```

Comment:
do people normally turn off lights when leaving their office to the break room? I'm slightly torn on this lol